charts/consul/templates/crd-meshes.yaml

```diff
@@ -66,10 +66,56 @@ spec:
               http:
                 description: HTTP defines the HTTP configuration for the service mesh.
                 properties:
+                  incoming:
```

Comment:
Docs here and below copied from `consul` changes. Subject to change based on feedback from Docs team.